ext/options.go

```diff
@@ -0,0 +1,50 @@
+package ext
+
+import (
+	"github.com/opentracing/opentracing-go"
+)
+
+// TODO move RPCServerOption into this file? OR, remove it. Code such as:
+//
+// err := tracer.Extract(...)
+// if err != nil { ... }
+// tracer.StartSpan("...", RPCServerOption(...))
+//
+// can be rewritten as:
+// tracer.StartSpan("...", SpanKindRPCServer, ChildOfCarrierOption(...))
```

Comment:
this comment is out-of-date naming wise, though I prefer `ChildOfCarrierOption` to `CarriedChildOfOption`... if you read it quickly, the latter sounds like the child is the thing in the carrier, not the parent (which is the case, of course).